src/tests/Navbar.test.js

```diff
@@ -0,0 +1,9 @@
+import renderer from 'react-test-renderer';
+import { BrowserRouter as Router } from 'react-router-dom';
+import Navbar from '../components/Navbar';
+
+it('renders correctly', () => {
+  const tree = renderer.create(<Router><Navbar /></Router>).toJSON();
+
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] Please use React testing library to create some integration tests that simulate user interaction

Reply:
Thanks for your suggestion, although the only way I've found to test link changes after a user clicks on the buttons (The only interaction possible with the Navbar) is creating a function inside the original Navbar, which would result in a linter error, since we won't use the mentioned function at all.

Another workaround is to create said function in a refactored Navbar, but we won't be testing link changes in the original in that case.